sdk/core/Azure.Core.TestFramework/src/RecordedTestSanitizer.cs

```diff
@@ -33,6 +36,11 @@ public virtual void SanitizeHeaders(IDictionary<string, string[]> headers)
 
         public virtual string SanitizeTextBody(string contentType, string body)
         {
+            foreach (var jsonPath in JsonPathSanitizers)
+            {
+                return SanitizeJsonBody(body, jsonPath);
```

Comment:
Will this run all sanitizers?  It looks like it returns after the first one, but it seems like we'd want to run all sanitizers in the collection?